### DISCUSSION TITLE
How to get subtitles on the PyTorch YouTube video?

### DISCUSSION THREAD

**Discussion by alirezaghanbari6** [mrdbourke/pytorch-deep-learning#79] [General]
Hi mrdbourke, can I ask why the subtitles are disabled on your YouTube channel?

2- How to implement a model for self-driving cars with pytorch library? Thank you 

Alireza Ghanbari

**Comment by mrdbourke** [comment]
Hey @alirezaghanbari6,

I haven't disabled subtitles, it may just be too long for YouTube to create them.

Otherwise if you're watching in Chrome you can enable automatic subtitles here: https://support.google.com/chrome/answer/10538231?hl=en 

As for implementing a model for self-driving cars, this is a huge research problem (possibly the biggest in machine learning right now) so for that I don't have an exact answer. Tesla are doing it with a vision-based model, you can see a video on how they're doing it here: https://www.youtube.com/watch?v=j0z4FweCy4M

**Reply by alirezaghanbari6** [reply]
Hello, dear Daniel, thank you for your good advice.
I had some work and did not see your message
thank you again .
I just want a reinforcement learning course from you.
Is it possible?

**Reply by mrdbourke** [reply]
I don't have much experience with reinforcement learning but if it's something I get into more, I'll potentially make a course.

**Reply by alirezaghanbari6** [reply]
Hello Dear Daniel:
  Thank you, how about software and hacking?
And our next question is:
How much ability should I have to apply to Canada or Europe?
And what should I do?
My third question is:
How well does data mining work for machine learning with real world data?
Thank you for your reply
